n4js-libs/packages/n4js-runtime-fetch/src/n4js/Headers.n4jsd

```diff
@@ -12,16 +12,16 @@
 
 import {IteratorExt} from "Iterator"
 
-export external public class ~Headers implements Iterable<Array<string>> {
+export external public class Headers implements Iterable<Iterable2<string, string>> {
```

Comment:
why change from `Array` to `Iterable2` when then using the same type of the first and later elements?

Reply:
Different API:
- `getAll()` returns all values of a particular key
- iterable/entries return tuples of key and single value; keys may appear multiple times